### ISSUE TITLE
[Suggestion] Save custom set controls upon closure of the game

### ISSUE THREAD

**Issue by Payday02** [tsunamayo/Skywanderers#166]
I’m used to using different controls when I play similar games, so I like to change some of the simple keybinds like making space bar go up and c go down. The problem is every time I close the game, the bindings reset to default and I have to do it again. It’s nothing more than a minor inconvenience but I thought I’d share it.

**Comment by tsunamayo** [comment]
It is not minor it is very painful! The settings are saved inside
C:\Users\{YOUR USER]\AppData\LocalLow\TsunamayoCorp\Skywanderers\SETTINGS.ini
Could you make sure that when your assignment is done this file is updated?
If not could you please check that the file and folder are not in read-only?
Thanks

**Comment by Payday02** [comment]
I think the issue stemmed from the fact that I have the game downloaded to my D: drive. The folder with the settings is in my C: drive so maybe it was not able to overwrite the document. I went on a quest through my system and found the correct folder in my C: drive I was able to manually edit the settings document and now it works fine. The game just doesn’t do it for me for some reason but I am able to adjust my settings manually.
